src/cmdlib.sh

```diff
@@ -267,6 +267,10 @@ commit_overlay() {
     name=$1
     path=$2
     respath=$(realpath "${path}")
+    # Only keep write bit for owner for all overlay files/dirs. This modifies
+    # the source config, but meh... git doesn't track it anyway and it's easier
+    # than using ostree's --statoverride when dealing with executable files.
+    chmod -R go-w "${respath}"
```

Comment:
should we make this verbose so we know which files get touched?

Reply:
Adding `-c, --changes` would be great indeed